garrysmod/gamemodes/terrortown/gamemode/lang/italian.lua

```diff
@@ -307,7 +307,7 @@ L.set_title_lang = "Impostazioni lingua"
 -- trovare l'impostazione della lingua anche se è una lingua che non conoscono.
 -- It may be best to leave this next one english, so english players can always
 -- find the language setting even if it's set to a language they don't know.
-L.set_lang = "Language selection:"
+L.set_lang = "Selezionare lingua (Select language):"

```

Comment:
My Italian is not fluent but the infinitive for selezionare seems weird here (vs imperative). I know that came up for Portuguese as well, and the right choice probably depends on the language. Maybe @PinoMartirio has an opinion?

Reply:
The right choice doesn't really depend on the language for Romance languages, it's apparently standard in the modern age for Spanish and Portuguese to use unconjugated forms instead of the singular they form when you don't have a specific target. I decided to do the same for Italian for consistency.